extensions/amp-script/0.1/amp-script.js

```diff
@@ -640,7 +649,8 @@ export class SanitizerImpl {
             dev().error(TAG, 'Invalid AMP.setState() argument: %s', value);
           });
           if (state) {
-            bind.setState(state, /* skipEval */ true, /* skipAmpState */ false);
+            const skipEval = !this.userActivationTracker_.isActive();
+            bind.setState(state, skipEval, /* skipAmpState */ false);
```

Comment:
Minor: Can we add a simple comment explaining this? E.g. "Only evaluate (mutate the DOM) if there was a recent user gesture.".

Reply:
Done